raiden/network/blockchain_service.py

```diff
@@ -26,6 +25,7 @@
     typecheck,
 )
 from raiden_contracts.contract_manager import ContractManager
+from raiden_contracts.utils.type_aliases import ChainID
```

Comment:
Can you import the contracts `ChainID` in the `raiden.utils.typing` module, so that imports like these don't change?

Reply:
Sure.  Will do this.